lib/tasks/npm-task.js

```diff
@@ -28,6 +29,10 @@ class NpmTask extends Task {
     this.versionConstraints = '3 || 4 || 5 || 6';
   }
 
+  get packageManagerOutputName() {
+    return this.packageManager.name === 'yarn' ? 'Yarn' : 'npm';
```

Comment:
Still seems like `pnpm` needs to be added here

Reply:
🤦 yep, missed that 😅